solution.py

```diff
@@ -1365,8 +1589,11 @@ def min_max_scale(image:ArrayLike)->ArrayLike:
 
         ###### Segmentation based metrics #########
         # Load the manually curated nuclei target label
-        print('Computing Segmentation Metrics')
-        pred_label,target_label= cellpose_segmentation(predicted_nuc_phase2fluor,target_nucleus)
+        pbar.set_description(f"Processing FOV {fov} - Computing Segmentation Metrics")
+        pbar.refresh()
+        pred_label, target_label = cellpose_segmentation(
+            predicted_nuc_phase2fluor, target_nucleus
```

Comment:
```suggestion
            predicted_nuc_phase2fluor, target_nucleus[0]
```
`target_nucleus` is 4-dimensional here (`(1,1,2048,2048)`), which tricks Cellpose into thinking this is a 3D stack and an error is raised.